tests/mock_tables/dbconnector.py

```diff
@@ -98,6 +105,20 @@ def __init__(self, *args, **kwargs):
                 for k, v in table.items():
                     self.hset(h, k, v)
 
+    def _encode(self, value):
+        "Return a bytestring representation of the value. Taken from redis-py connection.py"
+        if isinstance(value, bytes):
+            return value
+        elif isinstance(value, (int, long)):
+            value = str(value).encode('utf-8')
+        elif isinstance(value, float):
+            value = repr(value).encode('utf-8')
+        elif not isinstance(value, basestring):
+            value = str(value).encode('utf-8')
+        # else:
```

Comment:
is this else not required?

Reply:
Resolved with complete mock.

---
In reply to: [510372790](https://github.com/Azure/sonic-snmpagent/pull/171#discussion_r510372790) [](ancestors = 510372790)